lib/apiservers/engine/backends/system.go

```diff
@@ -262,6 +265,25 @@ func getImageCount(plClient *client.PortLayer) int {
 	return len(images)
 }
 
+// Use the Portlayer's support for docker ps to get the container count
 func getContainerStatus(plClient *client.PortLayer) (ContainerStatus, error) {
-	return ContainerStatus{0, 0, 0, 0}, nil
+	var status ContainerStatus
+
+	all := new(bool)
+	*all = true
+	containList, err := plClient.Containers.GetContainerList(containers.NewGetContainerListParams().WithAll(all))
+	if err != nil {
+		return status, derr.NewErrorWithStatusCode(fmt.Errorf("Failed to get container list: %s", err), http.StatusInternalServerError)
+	}
+
+	for _, t := range containList.Payload {
+		if *t.Status == "Running" {
+			status.numRunning++
+		} else if *t.Status == "Stopped" {
+			status.numStopped++
+		}
+		status.count++
```

Comment:
Do we care about paused containers? We'd be counting them as well here, although I'm not sure we care about them right now. `numPaused` in the `ContainerStatus` struct seems to suggest so.